src/ports/analyticsDayData/utils.ts

```diff
@@ -52,6 +52,19 @@ export function getAnalyticsTotalDataQuery() {
   `
 }
 
+export function mapAnalyticsFragment(fragment: any): AnalyticsDayData {
+  // The data returned from the collections/marketplace subgraphs is identical to the one that will be returned by the nft-server.
+  // This means that there is no need to map it, just return the same values.
+  return {
+    id: fragment.id,
+    date: fragment.date,
+    sales: fragment.sales,
+    volume: fragment.volume,
+    creatorsEarnings: fragment.creatorsEarnings,
+    daoEarnings: fragment.daoEarnings,
+  }
```

Comment:
If the type is exactly the same, what do you think about using an identity function `(value) => value` or making it possible to make this mapping function optional?